packages/autorest.python/autorest/codegen/serializers/__init__.py

```diff
@@ -495,6 +507,42 @@ def _serialize_and_write_metadata(
             namespace_path / Path("_metadata.json"), metadata_serializer.serialize()
         )
 
+    # find root folder where "setup.py" is
+    def _package_root_folder(self, namespace_path: Path) -> Path:
+        return namespace_path / Path("../" * (self.code_model.namespace.count(".") + 1))
+
+    def _serialize_and_write_sample(self, env: Environment, namespace_path: Path):
+        out_path = self._package_root_folder(namespace_path) / Path("generated_samples")
+        for op_group in self.code_model.operation_groups:
+            if self.code_model.options["multiapi"]:
```

Comment:
do we want generated samples for multiapi? Is there a real use case? I fear it will get too complicated too fast. Other languages don't have multiapi support so we don't need it for language compatibility

Reply:
@iscai-msft python has 30 multiapi packages. If we don't support multiapi, many packages (network, compute, etc) will be influenced.

Reply:
These samples aren't correct though, we should never direct people to go into individual api version folders, it should all be handled at the root multiapi client. We can do a separate pr for multiapi, but this won't generate properly

Reply:
@iscai-msft It is hard to pick the moderate api-version before we know all the api-versions. But after we find the default api-version, we can't get the example-file info...
Ok, let us focus on single-api package at first and I will think about how to generate sample for multiapi in another PR.